messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/TransportConfigOperationHandlers.java

```diff
@@ -340,16 +346,96 @@ static void processAcceptors(final OperationContext context, final Configuration
         configuration.setAcceptorConfigurations(new HashSet<>(acceptors.values()));
     }
 
+    public static final void processConnector(OperationContext context, ModelNode config) throws OperationFailedException, CapabilityServiceSupport.NoSuchCapabilityException {
+        String connectorType = context.getCurrentAddress().getLastElement().getKey();
+        CapabilityServiceBuilder builder = context.getCapabilityServiceTarget().addCapability(CONNECTOR_CAPABILITY);
+        builder.provides(CONNECTOR_CAPABILITY.getCapabilityServiceName(context.getCurrentAddress()));
+        builder.setInitialMode(ServiceController.Mode.ACTIVE);
+        switch (connectorType) {
+            case CommonAttributes.CONNECTOR: {
+                final Map<String, Object> parameters = getParameters(context, config, CONNECTORS_KEYS_MAP);
+                String socketBindingName = GenericTransportDefinition.SOCKET_BINDING.resolveModelAttribute(context, config).asStringOrNull();
+                final String clazz = FACTORY_CLASS.resolveModelAttribute(context, config).asString();
+                if (config.hasDefined(CommonAttributes.SSL_CONTEXT.getName())) {
+                    String sslContextName = config.get(CommonAttributes.SSL_CONTEXT.getName()).asString();
```

Comment:
The 'config' var is actual the 'operation' node from ConnectorAdd. We shouldn't directly use the operation node in Stage.RUNTIME.

It should use resolveModelAttribute as well.